_1327/polls/templates/polls_index.html

```diff
@@ -8,63 +8,105 @@
 {% endblock %}
 
 {% block content %}
-    <div class="panel panel-default">
-        <div class="panel-heading">
-            <div class="panel-title">
-                {% trans "Polls you can vote for" %}
-            </div>
-        </div>
-        {% if running_polls|length == 0 %}
-            <div class="panel-body">
-                <em>{% trans "There are no polls you can vote for." %}</em>
-            </div>
-        {% else %}
-            <table class="table table-striped">
-                <tr>
-                    <th>{% trans "Poll" %}</th>
-                    <th>{% trans "Open until" %}</th>
-                    <th></th>
-                </tr>
-                {% for poll in running_polls %}
-                    <tr>
-                        <td><a href="{% url "polls:vote" poll.id %}">{{ poll.title }}</a></td>
-                        <td>{{ poll.end_date }}</td>
-                        {% get_obj_perms request.user for poll as "poll_perms" %}
-                        {% if "change_poll" in poll_perms %}
-                            <td class="pull-right">
-                                <a class="btn btn-warning btn-xs" href="{% url "polls:edit" poll.id %}">{% trans "Change Poll" %}</a>
-                            </td>
-                        {% endif %}
-                    </tr>
-                {% endfor %}
-            </table>
-        {% endif %}
-    </div>
+	{% if upcoming_polls|length > 0 %}
+		<div class="panel panel-default">
+			<div class="panel-heading">
+				<div class="panel-title">
+					{% trans "Upcoming polls" %}
+				</div>
+			</div>
+			<table class="table table-striped">
+				<tr>
+					<th class="col-xs-6">{% trans "Poll" %}</th>
+					<th class="col-xs-5">{% trans "End Date" %}</th>
+					<th class="col-xs-1"></th>
+				</tr>
+				{% for poll in upcoming_polls %}
+					<tr>
+						<td>{{ poll.title }}</td>
+						<td>{{ poll.end_date }}</td>
+						{% get_obj_perms request.user for poll as "poll_perms" %}
+						{% if "change_poll" in poll_perms %}
+							<td class="pull-right">
+								<a class="btn btn-warning btn-xs" href="{% url "polls:edit" poll.id %}">{% trans "Change Poll" %}</a>
+							</td>
+						{% else %}
+							<td></td>
```

Comment:
does `class="pull-right"` have any impact on the empty `<td>`? Otherwise it would be nice to put the `if` only around the `a` without an else